transports/quic/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "libp2p-quic"
+version = "0.15.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "MIT"
+description = "A libp2p transport using QUIC"
+keywords = ["peer-to-peer", "libp2p", "quic", "networking"]
+categories = ["network-programming", "asynchronous"]
+
+[dependencies]
+rustls = { version = "0.16.0", features = ["dangerous_configuration"] }
+libp2p-core = { path = "../../core", version = "0.15.0" }
+log = "0.4.8"
+ipnet = "2.1.0"
+err-derive = "0.2.2"
+futures = "0.3.1"
+quinn-proto = { git = "https://github.com/djc/quinn" }
```

Comment:
This should be switched to a published version, otherwise we can't publish `libp2p-quic` on crates.io.

Reply:
`libp2p-quic` relies heavily on features that are only present in the current git master.  I could ask @djc to make a release.